commit 1db918c5966c6f95dda9b2c4c26975b0a8fddb2c
Author: Gabriel Scherer <gabriel.scherer@gmail.com>
Date:   Thu Aug 8 11:59:49 2013 +0200

    Add the new O_CLOEXEC flag to Unix.open_flag in version 4.1

diff --git a/src/batUnix.mliv b/src/batUnix.mliv
index bd710a7..943db0a 100644
--- a/src/batUnix.mliv
+++ b/src/batUnix.mliv
@@ -261,6 +261,10 @@ type open_flag = Unix.open_flag =
                                     O_SYNC/O_DSYNC) *)
 ##V4##  | O_SHARE_DELETE              (** OCaml 4 and Windows only: allow the file to be deleted
 ##V4##                                   while still open *)
+##V4.1##  | O_CLOEXEC                   (** Set the close-on-exec flag on the
+##V4.1##                                    descriptor returned by {!openfile}
+##V4.1##                                    
+##V4.1##                                    Since OCaml 4.1 *)
 (** The flags to {!Unix.openfile}. *)
 
 
